### PR TITLE
add gold analyze calendar

### PR DESCRIPTION
**Pull request by JakubSwiderskiPJ**

### CODE REVIEW

File: src/screens/gold/GoldAnalizeScreen.tsx [hunk 1]
```diff
@@ -1,16 +1,162 @@
...
+                Kliknij na dzień, aby zobaczyć szczegóły
+              </Text>
+            )}
+          </View>
+        </>
+      ) : null}
     </ScreenLayout>
   );
 };
+
```
Comment (OskarPrzydatek) [inline src/screens/gold/GoldAnalizeScreen.tsx:143]: doc here 😄

**Review by OskarPrzydatek — CHANGES_REQUESTED** [review]
Add for styles docs

File: src/screens/gold/GoldAnalizeScreen.tsx (hunk 1 above)
Comment (JakubSwiderskiPJ) [reply src/screens/gold/GoldAnalizeScreen.tsx:143]: Done 😃

**Review by OskarPrzydatek — APPROVED** [review]

**Review by OskarPrzydatek — APPROVED** [review]
LGTM 😎

**Pull request merged**